src/main/java/racinggame/domain/RacingCar.java

```diff
@@ -0,0 +1,39 @@
+package racinggame.domain;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class RacingCar {
+
+    private String carName;
+    private int carPosition = 0;
+
+    private RacingCar() {}
+
+    public RacingCar(String carName) {
+        this.carName = carName;
+    }
+
+    public static List<RacingCar> createRacingCarFromCarNames(String[] carNames) {
+        return Arrays.stream(carNames).map(carName -> new RacingCar(carName))
```

Comment:
[메서드 참조](https://jinseongsoft.tistory.com/208)를 통해 아래와 같이 작성이 가능합니다.
```suggestion
        return Arrays.stream(carNames).map(RacingCar::new)
```

Reply:
아직 이런게 익숙하지가 않나봅니다 ㅜㅜ 감사합니다!!